practiceCourseGenerator/practiceCourseGenerator.py

```diff
@@ -138,14 +143,21 @@ def populateUserDictiionary(userList, urlPrefix, urlPost):
 		userDictionaries.append(data)
 	return userDictionaries
 
+def generateMd5(fileName, fileNameBase):
+	checkSum = hashlib.md5(open(fileName, 'rb').read()).hexdigest()
+	logger.info('CheckSum for ' + str(fileName) + ': ' + str(checkSum))
+	#Write checkSum to file
+	checkSumFile = fileNameBase + 'MD5.txt'
+	with open(checkSumFile, 'wb') as writeFile:
+		writeFile.write(checkSum)
+
 def setupLogger(logdate):
 	#create logger 'canvasFileGenerator'
 	logger = logging.getLogger('canvasFileGenerator')
 	logger.setLevel(logging.INFO)
-	#logdate = time.strftime("%Y%m%d%H%M")
 
 	#create file handler
-	fh = logging.FileHandler('canvasFileGeneratorLog_' + logdate + '.log')
+	fh = logging.FileHandler(logDirectory + '/canvasFileGeneratorLog_' + logdate + '.log')
```

Comment:
Try to avoid global variables.  Pass logDirectory into setupLogger as an argument instead.